JavaScript Grocery List Project/js/app.js

```diff
@@ -100,7 +100,7 @@ function initList(){
             updateInner.addEventListener('click', editableItem);
             parent.appendChild(updateInner);
             
-            listHolder = listHolder.filter(item =>  item.itemContent == e.currentTarget.childNodes[0].childNodes[0].nodeValue );
+            listHolder = listHolder.filter(item =>  item.itemContent !== e.currentTarget.childNodes[0].childNodes[0].nodeValue );
             saveToLocal(listHolder);

```

Comment:
too much dot notation used  not need to use multiple notation for  just comparing value

Reply:
HI Hidayat Dada,
Now, I have set attribute data-item and getting value from it.
e.currentTarget.getAttribute("data-item");
Please have a look and let me know your thouths on it.